### DISCUSSION TITLE
can i parse the api with 115k data

### DISCUSSION THREAD

**Discussion by byCihat** [halaxa/json-machine#77] [Q&A]
sorry my english is a bit bad
I've been trying for 2 months but I couldn't make it

https://www.kupondas.xyz/oran/
 the data here comes as json using datables

api link total 115k data
https://www.kupondas.xyz/oran/scripts/ids-objects.php

Is it possible for me to parse this data quickly?

For example, if league is equal, if GUR and ms1 are equal, I need to parse like 3.00

i need serious help with this

Can you make a small sample if possible?

```
$json2 = file_get_contents('https://www.kupondas.xyz/oran/scripts/ids-objects.php');
$json_data = json_decode($json2,true);
foreach($json_data["data"] as $v){
 
    if($v["lig"]=="GUR" and $v["ms1"]=="3.00"){
        echo "<pre>";
        print_r($v);   
 
    }
}
```

**Comment by halaxa** [comment]
There's probably no need to use JSON Machine. 155k of data seems to be ok to use with `json_decode` unless you are getting some memory error. As far as the algorithm, a better place to get help might be StackOverflow. Unfortunately, it is out of the time scope I invest in this library.
